src/components/CharacterStats.tsx

```diff
@@ -449,6 +465,10 @@ export default CharacterStats;
 
 const styles = (theme: Theme) => css`
   .stats-controls {
+    &.no-stat-changes {
+      display: none;
+    }
+
```

Comment:
Instead of conditionally applying a class name that adds `display: none;`, can we just conditionally not render the entire `<div>`? It should be knowable at build time, and it shouldn't change.

Reply:
:skull: I forgor